src/blocks/catalog.scss

```diff
@@ -0,0 +1,23 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  padding: 50px 0;
+  column-gap: 48px;
+  row-gap: 46px;
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, auto);
+  }
```

Comment:
This property doesn't need a media query. When the viewport hits 488px, the next media query's property will overwrite this one.

Reply:
Right, removed.